assets/scss/base/_layouts.scss

```diff
@@ -10,34 +10,28 @@ body {
   counter-reset: section;
 }
 
+#wrapper {
+  text-align: left;
```

Comment:
used to override `govuk_elements` selector - will add issue to remove once `govuk_elements` has been removed